frontend/packages/knative-plugin/src/components/add/event-sources/EventSourceSection.tsx

```diff
@@ -19,50 +19,56 @@ import { isKnownEventSource } from '../../../utils/create-eventsources-utils';
 
 interface EventSourceSectionProps {
   namespace: string;
+  fullWidth?: boolean;
 }
 
-const EventSourceSection: React.FC<EventSourceSectionProps> = ({ namespace }) => {
+const EventSourceSection: React.FC<EventSourceSectionProps> = ({
+  namespace,
+  fullWidth = false,
+}) => {
   const { values } = useFormikContext<FormikValues>();
   const projectResource = { kind: ProjectModel.kind, prop: ProjectModel.id, isList: true };
   const [data, loaded] = useK8sWatchResource<K8sResourceKind[]>(projectResource);
   useFormikValidationFix(values);
-  if (!values.type) {
+  if (!values.formData.type) {
     return null;
   }
   let EventSource: React.ReactElement;
-  const sectionTitle = values.data?.itemData?.title ?? values.type;
-  switch (values.type) {
+  const sectionTitle = values.formData.data?.itemData?.title ?? values.formData.type;
+  switch (values.formData.type) {
     case EventSources.CronJobSource:
-      EventSource = <CronJobSection title={sectionTitle} />;
+      EventSource = <CronJobSection title={sectionTitle} fullWidth={fullWidth} />;
       break;
     case EventSources.SinkBinding:
-      EventSource = <SinkBindingSection title={sectionTitle} />;
+      EventSource = <SinkBindingSection title={sectionTitle} fullWidth={fullWidth} />;
       break;
     case EventSources.ApiServerSource:
-      EventSource = <ApiServerSection title={sectionTitle} />;
+      EventSource = <ApiServerSection title={sectionTitle} fullWidth={fullWidth} />;
       break;
     case EventSources.KafkaSource:
-      EventSource = <KafkaSourceSection title={sectionTitle} />;
+      EventSource = <KafkaSourceSection title={sectionTitle} fullWidth={fullWidth} />;
       break;
     case EventSources.ContainerSource:
-      EventSource = <ContainerSourceSection title={sectionTitle} />;
+      EventSource = <ContainerSourceSection title={sectionTitle} fullWidth={fullWidth} />;
       break;
     case EventSources.PingSource:
-      EventSource = <PingSourceSection title={sectionTitle} />;
+      EventSource = <PingSourceSection title={sectionTitle} fullWidth={fullWidth} />;
       break;
```

Comment:
I think we can refactor this to a util an it can be also reused in `CatalogEventSourceSection.tsx` as well, to avoid duplication of code.

Reply:
I kept it separate considering we can completely get rid of one file once we have Catalog support for sources

Reply:
Have updated and using only one component for both and removed `CatalogEventSourceSection.tsx`